src/wp-includes/class-wp-taxonomy.php

```diff
@@ -645,6 +645,7 @@ public static function get_default_labels() {
 				_x( 'A link to a tag.', 'navigation link block description' ),
 				_x( 'A link to a category.', 'navigation link block description' ),
 			),
+			'template_name'              => array( _x( 'Tag Archives', 'taxonomy template name' ), _x( 'Category Archives', 'taxonomy template name' ) ),
```

Comment:
```suggestion
```

This can be removed with the changes to `get_taxonomy_labels()`

Reply:
For some reason I assumed we wanted `get_default_labels()` to always return a default for all labels. If not, can we also remove the [default `template_name` label](https://github.com/WordPress/wordpress-develop/pull/6344/files#diff-52c8c412194f4e6bf199f6bbf576b66410eb470aa181330bdd7c290c9ed075e4R1013) from post type's `get_default_labels`, given that it's already set in `get_post_type_labels()`?

Reply:
I just removed it from both places in 48984658e90ce47b5b71892f588593e6a6d9460b.